cloud/filestore/libs/vfs_fuse/fs_impl_data.cpp

```diff
@@ -134,7 +135,8 @@ void TFileSystem::Release(
     if (Config->GetAsyncDestroyHandleEnabled()) {
         STORAGE_DEBUG("Add destroy handle request to queue #" << ino << " @" << fi->fh);
         with_lock(HandleOpsQueueLock) {
-            if (!HandleOpsQueue->AddDestroyRequest(ino, fi->fh)) {
+            const auto& res = HandleOpsQueue->AddDestroyRequest(ino, fi->fh);
+            if (res == 0) {
```

Comment:
let's add `enum class` to specify error kind in `AddDestroyRequest` returning -1,0,1 is really confusing